jetstream/cli.py

```diff
@@ -146,7 +163,9 @@ def execute(
                         spec.merge(external_spec)
 
                 config = spec.resolve(experiment, external_configs)
-                self.analysis_class(self.project_id, self.dataset_id, config).run(date)
+                analysis = self.analysis_class(self.project_id, self.dataset_id, config)
+                analysis.ensure_enrollments(date)
```

Comment:
Ensuring enrollments in the `SerialExecutorStrategy` and removing CLI calls.

Reply:
Hm, this is a precondition for Analysis.run(), right? We should either document that on the Analysis class or just move this call (sorry!) into Analysis.run, which ought to be safe for Argo now that we can call it separately.

Reply:
Indeed, good call! Moved into `Analysis.run()`